Userland/Libraries/LibWeb/Fetch/Body.cpp

```diff
@@ -183,15 +177,7 @@ WebIDL::ExceptionOr<JS::NonnullGCPtr<JS::Promise>> consume_body(JS::Realm& realm
     auto success_steps = [promise, &realm, &object, type](ByteBuffer const& data) {
         auto& vm = realm.vm();
 
-        // NOTE: Not part of the spec, but we need to have an execution context on the stack to call native functions.
-        //       (In this case, Promise's reject function and JSON.parse)
```

Comment:
This comment was providing useful information, please keep at least the mentioned functions

Reply:
Especially if we want to help fixing all of these eventually.